tests/baselines/reference/intersectionAndUnionTypes.errors.txt

```diff
@@ -118,7 +118,8 @@ tests/cases/conformance/types/intersection/intersectionAndUnionTypes.ts(37,1): e
     ~
 !!! error TS2322: Type 'A & B' is not assignable to type '(A & C) | (A & D) | (B & C) | (B & D)'.
 !!! error TS2322:   Type 'A & B' is not assignable to type 'B & D'.
-!!! error TS2322:     Type 'A & B' is not assignable to type 'D'.
+!!! error TS2322:     Property 'd' is missing in type 'A & B' but required in type 'D'.
+!!! related TS2728 tests/cases/conformance/types/intersection/intersectionAndUnionTypes.ts:4:15: 'd' is declared here.

```

Comment:
Interesting but not awful that we lose the line `Type 'A & B' is not assignable to type 'D'.`... and it’s nice that we get the elaboration into the missing properties.